### PR TITLE
generateLeagueTable 

### PR DESCRIPTION
**Pull request by ASP73**
adds generateLeagueTable function to utils.js, generateLeagueTableFunction test file to utils.test.js, updates fixtures.js test data to remove invalid venues

### CODE REVIEW

File: back-end/__test__/utils.test.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 const {
+    generateLeagueTable
```
Comment (HanzoDaBoss) [inline back-end/__test__/utils.test.js]: Need to add comma here

File: back-end/__test__/utils.test.js [hunk 2]
```diff
@@ -486,3 +487,139 @@ describe("Tests for compareMoney", () => {
...
+        {
+          team1_id: 1,
+          team2_id: 2,
+          team1_score: 2,
+          team2_score: 1,
+        },
+      ],
+    };
+
+    const teamStats = generateTableFunction(completedMatches);
```
Comment (HanzoDaBoss) [inline back-end/__test__/utils.test.js]: generateTableFunction here is not imported as a function from the start (There is generatLeagueTable imported at the start instead which isnt being used) ---> might need to look at this

**Review by HanzoDaBoss — COMMENTED** [review]
Good job Alec, well done on the tests, just need to fix some errors in order to run them :)
Also need to add jest and other dependencies in package.json

File: back-end/__test__/utils.test.js (hunk 1 above)
Comment (ASP73) [reply back-end/__test__/utils.test.js]: comma added

File: back-end/__test__/utils.test.js (hunk 2 above)
Comment (ASP73) [reply back-end/__test__/utils.test.js]: changed function name in test to match import which is the correct name

**Pull request closed without merging**